document-store/src/main/java/org/hypertrace/core/documentstore/parser/SortTypeExpressionVisitor.java

```diff
@@ -4,7 +4,7 @@
 import org.hypertrace.core.documentstore.expression.impl.FunctionExpression;
 import org.hypertrace.core.documentstore.expression.impl.IdentifierExpression;
 
-public interface SortingExpressionVisitor {
+public interface SortTypeExpressionVisitor {
```

Comment:
`SortExpressionVisitor` for easy readability?

Reply:
So in the visitor interfaces, I wanted to maintain naming consistency by just appending the `Visitor` keyword to the various types defined earlier to denote that any implementation is capable of visiting expression of that particular type.

Reply:
Sure. That's perfect. Whatever you name the base expression interfaces, keep the visitors consistent with that name.